app/serializers/show_book_serializer.rb

```diff
@@ -0,0 +1,11 @@
+class ShowBookSerializer < ActiveModel::Serializer
+  attributes :id, :author, :title, :genre, :publisher, :year, :image_url, :actual_rent
+
+  def image_url
+    object.image
+  end
+
+  def actual_rent
+    RentSerializer.new(object.rents.last).as_json
```

Comment:
Does this work if the book has no rents?

Reply:
Sorry, I will check it.